display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt

```diff
@@ -62,12 +67,17 @@ fun MainScreen(sampleName: String, application: Application) {
                 horizontalAlignment = Alignment.CenterHorizontally,
                 verticalArrangement = Arrangement.Center
             ) {
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier
                         .fillMaxSize()
                         .weight(1f),
-                    mapViewModel = mapViewModel
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(
+                        viewpoint = Viewpoint(39.8, -98.6, 10e7)
+                    ),
+                    mapViewProxy = mapViewModel.mapViewProxy,
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    onPan = { mapViewModel.dismissBottomSheet() }
```

Comment:
Maybe the following gesture events should also be overridden like the `onPan` event to dismiss the bottom sheet?

- `onDoubleTap`
- `onLongPress`
- `onTwoPointerTap`

Reply:
@XuemingRocks I appreciate your feedback, it has prompted some thoughtful consideration on my part.

During the implementation of this sample during the past release, there was a consideration regarding the interaction between the bottom sheet and map usage. It was suggested that allowing the dismissal of the bottom sheet upon initiating map interactions, such as panning to explore and identify additional features, would enhance visibility and prioritize the map's centrality.

Considering this, modifying other gestures to facilitate the dismissal of the bottom sheet would not notably improve the current functionality or user experience. There is also negligible likelihood of users using those specific events to dismiss the bottom sheet, as there is already a close button available on the bottom sheet for the same.

Reply:
Fair enough